apps/dashboard/app/views/layouts/application.html.erb

```diff
@@ -28,18 +28,14 @@
 <body>
   <header>
     <nav class="navbar navbar-expand-md shadow-sm navbar-color navbar-<%= @user_configuration.navbar_type %>">
-    <% if @user_configuration.dashboard_header_img_logo %>
-      <a class="navbar-brand navbar-brand-logo" href="<%= root_path %>"><img class="img-fluid" src="<%= @user_configuration.dashboard_header_img_logo %>" alt="<%= @user_configuration.dashboard_title %>"></a>
-    <% else %>
-      <a class="navbar-brand" href="<%= root_path %>"><%= @user_configuration.dashboard_title %></a>
-    <% end %>
+      <ul class="navbar-nav w-100" role="menubar">
```

Comment:
Similar to the helpbar, to vertically center the logo:
`<ul class="navbar-nav w-100 align-items-center" role="menubar">`